quantum/q_fourier_transform.py

```diff
@@ -90,6 +90,9 @@ def quantum_fourier_transform(number_of_qubits: int = 3) -> qiskit.result.counts
 
 
 if __name__ == "__main__":
+    import doctest
```

Comment:
We need to add some of the new tests here, rather than adding a import statement!!